### PR TITLE
K8s docs updates

### LINKED ISSUE #4344: k8s docs - import k8s docs markup

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- based on #4391
- updated docs with cleaned up markdown
- initial pass at design updates
- added prism syntax highlighting
- added x scrolling for code blocks

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/kubernetes/docs](http://0.0.0.0:8001/kubernetes/docs)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)

## Issue / Card

Fixes #4344

### CODE REVIEW

File: templates/kubernetes/docs/backups.md [hunk 1]
```diff
@@ -1,69 +1,59 @@
...
- currently supported by native Kubernetes. Various third party solutions are available -
- please refer to their own documentation for details.
+Backing up application specific data, normally stored in a persistent volume, is not currently supported by native Kubernetes. Various third party solutions are available -
+please refer to their own documentation for details.
 
- ## Creating an **etcd** snapshot
+## Creating an **etcd** snapshot
 
- <div class="p-notification--warning"><p class="p-notification__response">
+ <div class="p-notification--warning"><p markdown="1" class="p-notification__response">
```
Comment (anthonydillon) [inline templates/kubernetes/docs/backups.md]: What does `markdown="1"` do?

File: templates/kubernetes/docs/base_docs.html [hunk 2]
```diff
@@ -4,6 +4,10 @@
 
 {% block meta_copydoc %}https://drive.google.com/drive/folders/1FyMcu6vqGnyQMdWLRjvxJVWm8gHgslk4{% endblock meta_copydoc %}
 
+{% block head_extra %}
+<link rel="stylesheet" href="/static/css/prism.css" /><script src="/static/js/prism.js"></script>
```
Comment (anthonydillon) [inline templates/kubernetes/docs/base_docs.html]: Can you put these two imports on different lines please. Easy to see whats going on

File: templates/kubernetes/docs/troubleshooting.md [hunk 3]
```diff
@@ -144,93 +120,89 @@ Happy Helming!
...
+```bash
+juju show-status kubernetes-master --format json | jq --raw-output '.applications."kubernetes-master".units | keys[]'
+54.210.100.102
+```
 
-   ```bash
-   juju show-status kubernetes-master --format json | jq --raw-output '.applications."kubernetes-master".units | keys[]'
-   54.210.100.102
-   ```
+### 1. Update the kubeconfig file
```
Comment (anthonydillon) [inline templates/kubernetes/docs/troubleshooting.md]: Should this be 3?

File: templates/kubernetes/docs/backups.md (hunk 1 above)
Comment (pmahnke) [reply templates/kubernetes/docs/backups.md]: it tells the parser to parse the text inside the block

File: static/sass/_layout_documentation.scss [hunk 4]
```diff
@@ -5,6 +5,26 @@
...
+    .p-notification--information,
+    .p-notification--negative,
+    .p-notification--positive {
+      margin-left: 0.1rem;
+      margin-right: 0.1rem;
+    }
+
+    pre {
+      overflow-x: auto;
+      width: 700px;
```
[the remark this answers is not in the document]
Comment (pmahnke) [reply static/sass/_layout_documentation.scss]: ah, trying 100% didn't ever enable the overflow-x scroll to work

**Comment by tbille** [comment]
@pmahnke @anthonydillon my fault I am fixing it

File: templates/kubernetes/docs/backups.md (hunk 1 above)
Comment (pmahnke) [reply templates/kubernetes/docs/backups.md]: not sure it works

File: static/sass/_layout_documentation.scss [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+        border-right: 0;
+        flex: 1;
+        flex-direction: column;
+      }
+    }
+
+    &__content {
+      flex: 5;
+      margin: 2rem 0;
+      padding-left: 5%;
```
Comment (anthonydillon) [inline static/sass/_layout_documentation.scss]: Can you make this only large screens please

Comment (anthonydillon) [reply static/sass/_layout_documentation.scss]: And the margin

Comment (pmahnke) [reply static/sass/_layout_documentation.scss]: done

**Comment by nottrobin** [comment]
Fixes https://github.com/canonical-websites/www.ubuntu.com/issues/4388

**Pull request merged**